addon/components/rdf-input-fields/heading.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@glimmer/component';
+
+export default class RdfHeadingComponent extends Component {
+  get titleSkin() {
```

Comment:
Nitpick: I would rename this to `skin` since the title part is implicit in this component. (same for the level).